### PR TITLE
Integrated Mayhem

### PR DESCRIPTION
**Pull request by bmwtsn098**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    DEBIAN_FRONTEND=noninteractive apt-get install -y build-essential
+
+## Add Source Code
+ADD . /cascadia
+WORKDIR /cascadia
+
+## Build Step
+RUN go mod init cascadia
+RUN go mod tidy
+RUN go build
```
Comment (xansec) [inline Dockerfile:15]: Can we add a package stage here?

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Namespaced project name that the target belongs to
+project: bmwtsn098/cascadia-test
```
Comment (xansec) [inline Mayhemfile:2]: You can use a generic project name.

File: Mayhemfile [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+# Target name (should be unique within the project)
+target: cascadia-test
+
+# Base image to run the binary in.
+image: ghcr.io/bmwtsn098/cascadia:latest
```
Comment (xansec) [inline Mayhemfile:8]: And omit the image name.

**Review by xansec — CHANGES_REQUESTED** [review]

**Comment by bmwtsn098** [comment]
I've made the requested changes. Let me know if everything looks good. Thanks

**Comment by xansec** [comment]
LGTM!

**Pull request merged**